dockerfiles/services/Makefile

```diff
@@ -1,8 +1,15 @@
 Q := @
 REDIS_IMAGE := datadog/dd-trace-ci:php-redis-5.0
+REQUEST_REPLAYER_IMAGE := datadog/dd-trace-ci:php-request-replayer-1.0
 
 redis_build:
 	$(Q) docker build -t $(REDIS_IMAGE) redis
 
 redis_publish: redis_build
 	$(Q) docker push $(REDIS_IMAGE)
+
+request-replayer_build:
+	$(Q) docker build -t $(REQUEST_REPLAYER_IMAGE) request-replayer
+
+request-replayer_publish: request-replayer_build
+	$(Q) docker push $(REQUEST_REPLAYER_IMAGE)
```

Comment:
I know this was already here, but my personal preference is to manage Dockerfiles with a `docker-compose.yml` instead of a `Makefile`. Of course, that's a totally optional suggestion. 🙂